### ISSUE TITLE
HTML web storage

### ISSUE THREAD

**Issue by samoser** [oetiker/callbackery#32]
Would you please implement the [qooxdoo HTML web storage](http://www.qooxdoo.org/current/api/#qx.bom.Storage)?

**Comment by oetiker** [comment]
for ticket #37 you mean ?

**Comment by samoser** [comment]
Yes, and initially for the language switch as well. But that is actually not needed.
Thanks.
